KomodoPlatform/atomicDEX-Desktop
# Change checkbox style

Requested by whitelabel project.

Before:
![image](https://user-images.githubusercontent.com/35845239/205974837-7aa684ca-5272-407e-905e-f2ae13c1f28e.png)

After:
![image](https://user-images.githubusercontent.com/35845239/205974646-d791cbfc-a334-4a7c-8cf0-5b398046fe85.png)

To Test:
- Check all checkboxes are displayed as the new style
- Confirm checkbox size is appropriate next it's label
- Confirm no label overflow etc as result of change
